crawl/pom.py

```diff
@@ -322,10 +333,19 @@ def _process_pom_template_content(self, pom_template_content):
         Returns a tuple:
            (updated_pom_template_content, pomparser.ParsedDependencies instance)
         """
```

Comment:
Would it make sense to split the property handling out into a different method for better readability?